components/tags/src/vwc-tag.scss

```diff
@@ -83,6 +83,7 @@
 
 	&.vwc-tag--selected {
 		&.layout-outlined {
+			border-color: var(#{layout.$vvd-layout-color-outline});
```

Comment:
according to the figma it is not the right color:
![image](https://user-images.githubusercontent.com/7512792/168586083-28bdc7a2-b262-45a8-ab66-01436574b331.png)
and in the code:
![image](https://user-images.githubusercontent.com/7512792/168586138-0c2a6642-b8c4-4355-a3ed-9cfad71fd03e.png)

Reply:
given that this is in discussions with design and partially handled in vonage/vivid-3#389
can this be resolved for now?